get/all.go

```diff
@@ -0,0 +1,173 @@
+package get
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/ninech/apis"
+	management "github.com/ninech/apis/management/v1alpha1"
+	meta "github.com/ninech/apis/meta/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/internal/format"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type allCmd struct {
+	out                  io.Writer
+	stdErr               io.Writer
+	IncludeNineResources bool `arg:"" help:"show resources which are owned by Nine" default:"false"`
+}
+
+type projectContentOptions struct {
+	out    io.Writer
+	output output
+}
+
+func (cmd *allCmd) Run(ctx context.Context, client *api.Client, get *Cmd) error {
+	projectName := client.Project
+	if get.AllProjects {
+		projectName = ""
+	}
+	projectList, err := projects(ctx, client, projectName)
+	if err != nil {
+		return err
+	}
+
+	items, warnings, err := getProjectContent(ctx, client, projectNames(projectList), cmd.IncludeNineResources)
+	if err != nil {
+		return err
+	}
+	// we now print all warnings to stderr
+	for _, w := range warnings {
+		fmt.Fprintf(defaultStdError(cmd.stdErr), "warning: %s\n", w)
+	}
+
+	if len(items) == 0 {
+		printEmptyMessage(cmd.out, "Resource", projectName)
+		return nil
+	}
+
+	switch get.Output {
+	case full:
+		return printItems(items, *get, defaultOut(cmd.out), true)
+	case noHeader:
+		return printItems(items, *get, defaultOut(cmd.out), false)
+	case yamlOut:
+		return format.PrettyPrintObjects(items, format.PrintOpts{Out: cmd.out})
+	}
+
+	return nil
+}
+
+func projectNames(projects []management.Project) []string {
+	result := make([]string, len(projects))
+	for i, proj := range projects {
+		result[i] = proj.Name
+	}
+
+	return result
+}
+
+func getProjectContent(ctx context.Context, client *api.Client, projNames []string, includeNineOwned bool) ([]*unstructured.Unstructured, []string, error) {
+	var warnings []string
+	var result []*unstructured.Unstructured
+	scheme := runtime.NewScheme()
+	if err := apis.AddToScheme(scheme); err != nil {
+		return nil, nil, err
+	}
```

Comment:
I guess we could also use the existing `client.Scheme()` as it already contains the all `apis` schemes and we filter by nine types anyways.

Reply:
changed this to use the existing client.Scheme